CMSgov/design-system
# [WNMGDS-888] Update beta analytics help drawer

## Summary
To prepare for a release of the beta Help drawer component with GA event tracking as a POC, Blast suggested these key-value pairs changes.
```
Open help drawer event:
    event_name: 'help_drawer_opened',
    event_type: 'ui interaction',
    ga_eventAction: 'opened help drawer',
    ga_eventCategory: 'ui components',
    ga_eventLabel: heading,
    ga_eventType: 'cmsds',
    ga_eventValue: '', 
    heading: heading,

Close help drawer event:
    event_name: 'help_drawer_closed',
    event_type: 'ui interaction',
    ga_eventAction: 'closed help drawer',
    ga_eventCategory: 'ui components',
    ga_eventLabel: heading,
    ga_eventType: 'cmsds',
    ga_eventValue: '', 
    heading: heading
```

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-888/update-beta-analytics-helpdrawer/components/help-drawer/

Test locally:
Test that Help drawer component works as per normal. Then proceed to test GA event tracking.

- Go to `Help Drawer` component, the React example and click `Toggle the helpdrawer` to open help drawer.
- Inspect DevTools network tab and filter for `collect` to view the analytics data or view using Chrome's extension `Omnibug`
- `Close` help drawer and inspect DevTools network tab to view the analytics data again
   -  note that the network data only shows the 3 values. Other values are there to support other items/future GA4 implementation
   
![Screen Shot 2021-04-20 at 4 42 30 PM](https://user-images.githubusercontent.com/20322880/115586182-520a4480-a29a-11eb-8e44-3f75a7d46ce7.png)
